### PR TITLE
am gov recipe

### PR DESCRIPTION
**Pull request by brittany-johnson**
https://github.com/openstax/cnx-recipes/issues/3506

### CODE REVIEW

**Review by LostMojo — APPROVED** [review]
oooh, it's so _clean_

**Review by kswanson33 — CHANGES_REQUESTED** [review]
This needs specs! Look at chemistry, uphysics, and especially the documentation Karina's adding in the [calculus PR](https://github.com/openstax/recipes/pull/27/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R214-R231) for an example.

Aside from that, it looks good!

File: books/american_government/bake [hunk 1]
```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'bundler/inline'
+
+gemfile do
+  gem 'openstax_kitchen', ENV['USE_LOCAL_KITCHEN'] ? { path: '/code/kitchen' } : '2.0.0'
```
Comment (jpslav) [inline books/american_government/bake:8]: are you still using `2.0.0` locally?  If not, should update this to match.

File: books/american_government/bake [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+      BakeExample.v1(example: example,
+                     number: "#{appendix_letter}#{example.count_in(:page)}",
+                     title_tag: 'div')
+    end
+
+    BakeAppendix.v1(page: page, number: appendix_letter)
+  end
+
+  # Bake titled and unclassified notes
+  book.chapters.pages.notes.each do |note|
```
Comment (jpslav) [inline books/american_government/bake:79]: do we need `chapters.pages` here?  can it be `book.notes`? Doesn't seem that `BakeUnclassifiedNotes` needs the chapter or page.

Actually, the latest interface of `BakeUnclassifiedNote` just takes the `book` and already includes the `note.classes.empty?` check -- can you upgrade and use that?

File: books/american_government/bake [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    note.classes.empty? && BakeUnclassifiedNote.v1(note: note)
```
Comment (jpslav) [inline books/american_government/bake:80]: Generally speaking I'm not a big fan of this kind of use of `&&`.  Would rather change

```ruby
condition && thing_to_do
```

to

```ruby
thing_to_do if condition
```

reads better IMO

**Review by jpslav — CHANGES_REQUESTED** [review]

**Review by kswanson33 — APPROVED** [review]

File: books/american_government/bake [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+  BakeChapterTitle.v1(book: book)
+  BakeChapterIntroductions.v1(book: book)
+  BakeUnclassifiedNotes.v1(book: book)
+
+  book.chapters.each do |chapter|
+    BakeChapterGlossary.v1(chapter: chapter, metadata_source: metadata)
+    BakeChapterSummary.v1(chapter: chapter, metadata_source: metadata)
+    BakeNonIntroductionPages.v1(chapter: chapter)
+
+    chapter.search('.review-questions, .critical-thinking').exercises.each do |exercise|
```
Comment (jpslav) [inline books/american_government/bake:33]: Can these have the element name?  Element name searches are faster.  E.g. if they are always `div`...

```ruby
chapter.search(%w[review-questions critical-thinking].prefix('div.'))
```

Karina added the `prefix` method

... actually should have a line that is 

```ruby
exercise_section_classes = %w[review-questions critical-thinking]
```

and use that

```ruby
chapter.search(exercise_section_classes.prefix('div.'))
```

to help with below

File: books/american_government/bake [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+    MoveExercisesToEOC.v1(
+      chapter: chapter,
+      metadata_source: metadata,
+      klass: 'review-questions'
+    )
+    MoveExercisesToEOC.v1(
+      chapter: chapter,
+      metadata_source: metadata,
+      klass: 'critical-thinking'
+    )
```
Comment (jpslav) [inline books/american_government/bake:46]: Some repeated code here.  Consider replacing with

```ruby
exercise_section_classes.each do |klass|
  MoveExercisesToEOC.v1(chapter: chapter, metadata_source: metadata, klass: klass)
end
```

File: books/american_government/bake [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+
+  book.pages('$.appendix').each do |page|
+    appendix_letter = [*('A'..'Z')][page.count_in(:book) - 1]
+
+    page.figures.each do |figure|
+      BakeFigure.v1(figure: figure, number: "#{appendix_letter}#{figure.count_in(:page)}")
+    end
+
+    page.tables('$:not(.unnumbered)').each do |table|
+      BakeNumberedTable.v1(table: table, number: "#{appendix_letter}#{table.count_in(:page)}", always_caption: true)
```
Comment (jpslav) [inline books/american_government/bake:67]: this line kinda long

File: books/american_government/bake [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+    MoveSolutionsToAnswerKey.v1(
+      chapter: chapter,
+      metadata_source: metadata,
+      strategy: :default,
+      strategy_options: {
+        selectors: ['section.review-questions']
+      },
+      append_to: solutions_container
+    )
+  end
```
Comment (jpslav) [inline books/american_government/bake:99]: does this `MoveSolutions...` call need to be here?  Can it instead be at the end of the other `book.chapters` loop above?

**Review by jpslav — CHANGES_REQUESTED** [review]

**Review by jpslav — APPROVED** [review]

**Pull request merged**